packages/@angular/cli/commands/generate.ts

```diff
@@ -126,6 +126,17 @@ export default Command.extend({
 
     return blueprint.install(blueprintOptions)
       .then(() => {
+        const VersionCheckTask = require('../tasks/version-check').default;
```

Comment:
This should happen before the command is run. And we could separate the command output with an empty line.